app/scripts/modules/kubernetes/job/configure/CommandBuilder.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.kubernetes.jobCommandBuilder.service', [
+  require('../../../core/config/settings.js'),
+  require('../../../core/account/account.service.js'),
+  require('../../../core/naming/naming.service.js'),
+  require('../../../core/utils/lodash.js'),
+  require('../../cluster/cluster.kubernetes.module.js'),
+])
+  .factory('kubernetesJobCommandBuilder', function (settings, $q, accountService, namingService, _,
```

Comment:
`settings`, `accountService`, `namingService`, `_` are all unused and can be removed.